sdk/lib/opentelemetry/sdk/trace/samplers/decision.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Trace
+      module Samplers
+        # The Decision class represents an arbitrary sampling decision. It can
```

Comment:
This comment is no longer accurate. `Decision` is now a module an enumeration of constants.